src/zenml/zen_server/deploy/helm/templates/server-deployment.yaml

```diff
@@ -91,6 +91,10 @@ spec:
               value: {{ $value | quote }}
             {{- end }}
             {{- end }}
+            {{- if .Values.zenml.baseURL }}
```

Comment:
Same here for the metadata? Or is this something that's supposed to be a little more hidden that we only use for cloud tenants?

Reply:
Yes, that was my intention. I have kept it out of the helm chart for now but we can add it in if there's a need.